pkg/proto/configuration/bb_storage/bb_storage.proto

```diff
@@ -89,6 +90,15 @@ message ApplicationConfiguration {
   // operation. This is hopefully safe, as operation names are hard to guess,
   // and the forwarded-to scheduler should perform its own authorization.
   buildbarn.configuration.auth.AuthorizerConfiguration execute_authorizer = 16;
+
+  // List of compression algorithms supported by the Content Addressable
+  // Storage. Valid values include:
```

Comment:
Sorry for coming back to this, but maybe we should extend this description a bit?

> List of compression algorithms supported by the Content Addressable Storage **to announce as part of the server's cache capabilities**.

It's probably also worth mentioning that this option does not actually control which compression algorithms are implemented. Clients are always permitted to issue requests using Zstandard. It's merely that not announcing it will cause clients to not use it.

Reply:
Let me know how the rewording looks, happy to iterate here for clarity.